restaurant-recommendation.ipynb

```diff
@@ -607,76 +607,381 @@
+       "3  22.149708800000003  -100.97609279999999   \n",
+       "4  22.149708800000003  -100.97609279999999   \n",
+       "\n",
+       "                                      the_geom_meter  \\\n",
+       "0  0101000020957F0000A6BF695F136F5AC1DADF87B20556...   \n",
+       "1  0101000020957F0000A6BF695F136F5AC1DADF87B20556...   \n",
+       "2  0101000020957F0000A6BF695F136F5AC1DADF87B20556...   \n",
+       "3  0101000020957F0000649D6F21634858C119AE9BF528A3...   \n",
+       "4  0101000020957F0000649D6F21634858C119AE9BF528A3...   \n",
+       "\n",
+       "                         name  ...   price url Rambience franchise    area  \\\n",
+       "0                   Paniroles  ...  medium   ?     quiet         f  closed   \n",
+       "1                   Paniroles  ...  medium   ?     quiet         f  closed   \n",
+       "2                   Paniroles  ...  medium   ?     quiet         f  closed   \n",
+       "3  El Rinc�n de San Francisco  ...  medium   ?  familiar         f    open   \n",
+       "4  El Rinc�n de San Francisco  ...  medium   ?  familiar         f    open   \n",
+       "\n",
+       "  other_services                                             userID  \\\n",
+       "0       Internet                            U1030,U1020,U1051,U1041   \n",
+       "1       Internet                            U1030,U1020,U1051,U1041   \n",
+       "2       Internet                            U1030,U1020,U1051,U1041   \n",
+       "3           none  U1055,U1126,U1024,U1004,U1106,U1016,U1101,U113...   \n",
+       "4           none  U1055,U1126,U1024,U1004,U1106,U1016,U1101,U113...   \n",
+       "\n",
+       "                rating          food_rating       service_rating  \n",
+       "0              0,2,1,1              0,2,1,2              0,1,1,1  \n",
+       "1              0,2,1,1              0,2,1,2              0,1,1,1  \n",
+       "2              0,2,1,1              0,2,1,2              0,1,1,1  \n",
+       "3  2,2,1,2,0,2,0,0,2,1  2,2,1,2,0,2,0,0,2,1  2,2,1,2,0,2,0,0,2,1  \n",
+       "4  2,2,1,2,0,2,0,0,2,1  2,2,1,2,0,2,0,0,2,1  2,2,1,2,0,2,0,0,2,1  \n",
+       "\n",
+       "[5 rows x 30 columns]"
+      ]
+     },
+     "execution_count": 69,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# Step 1: Combine all the data"
+    "# Step 1: Combine all the data\n",
+    "# Map all data to strings so it can be merged\n",
+    "ratings = rating_final.applymap(str)\n",
+    "accepts = accepts.applymap(str)\n",
+    "cuisine = cuisine.applymap(str)\n",
+    "hours = hours.applymap(str)\n",
+    "parking = parking.applymap(str)\n",
+    "geoplaces = geoplaces.applymap(str)\n",
+    "\n",
+    "# Group ratings\n",
+    "ratings = ratings.groupby(['placeID']).agg({\"userID\": ','.join, \"rating\": ','.join, \"food_rating\": ','.join, \"service_rating\": ','.join}).reset_index()\n",
+    "\n",
+    "# Merge tables\n",
+    "all = accepts.merge(cuisine, on='placeID')\n",
```

Comment:
We don't care about what the place accepts or the hours of the place. I like thought about what I care about when I get recommended a restaurant and all I really care about is where it is, is parking going to be a big hassle, the type of food, and where it is.